### DISCUSSION TITLE
How are different Databases analysed ?

### DISCUSSION THREAD

**Discussion by clojj** [azimuttapp/azimutt#111] [General]
Hi,
seems that I can't find infos about how Azimutt parses/analyses different databases.
Does the server provide generic database infos ? Is a database connection involved at all ?

Best, Joerg

**Comment by loicknuchel** [comment]
Hi @clojj, thanks for asking!

Azimutt analyze databases in the generic way, no specific analysis for now.
First, you load your database schema into Azimutt (using SQL script or database connection) and then Azimutt can make recommendations regarding your database schema (identify code smells).
Is it clearer?
